pkg/types/builder_test.go

```diff
@@ -180,7 +180,7 @@ func TestBuilder_generateTypeName(t *testing.T) {
 			g := &Builder{
 				Package: p,
 			}
-			got, gotErr := generateTypeName(tc.args.suffix, g.Package, tc.args.names...)
+			got, gotErr := generateTypeName(tc.args.suffix, g.Package, map[string]string{}, tc.args.names...)
```

Comment:
Could you please add a test case checking the override functionality?

Reply:
Done!